src/mlpack/tests/q_learning_test.cpp

```diff
@@ -49,7 +49,7 @@ BOOST_AUTO_TEST_CASE(CartPoleWithDQN)
   model.Add<Linear<>>(128, 2);
 
   // Set up the policy and replay method.
-  GreedyPolicy<CartPole> policy(1.0, 1000, 0.1);
+  GreedyPolicy<CartPole> policy(1.0, 1000, 0.1, 0.99);
```

Comment:
Haven't tested it but does this has any visible effect on the test results?

Reply:
@zoq Yes by adding this the change in parameters as they approach the final result is reducing as the iteration proceeds.